Automattic/wp-calypso
# Jetpack onboarding: add dark color scheme to onboarding flow

#### Changes proposed in this Pull Request

* Jetpack onboarding: add color scheme to onboarding flow.
* This covers the following existing steps of our new flow:
  * Login/Account creation
  * Connection/connection approval
  * Site type
  * Site topic
  * Plan selection

#### Testing instructions

* Fire up this branch locally.
* Spin up a new [JN site with these settings](https://jurassic.ninja/create/?shortlived&jetpack-beta).
* Copy the URL from the `Set up Jetpack` button and append `&calypso_env=development`.
* Make sure every step looks good on all devices.

#### Notes

- This wasn't tested with other color schemes other than Classic Bright and Classic Blue.
- I've overridden the colors in `WpcomColophon` using styles. Do we want to pass a className to it instead? cc @tyxla 

#### Before

![typical-urial-173 jurassic ninja-01](https://user-images.githubusercontent.com/390760/53639456-33134900-3c21-11e9-93bb-82d975fe9ed1.png)
![typical-urial-173 jurassic ninja-02](https://user-images.githubusercontent.com/390760/53639457-33134900-3c21-11e9-9789-80da73d7eecd.png)
![typical-urial-173 jurassic ninja-03](https://user-images.githubusercontent.com/390760/53639458-33abdf80-3c21-11e9-8be1-d2cf84644c49.png)
![typical-urial-173 jurassic ninja-04](https://user-images.githubusercontent.com/390760/53639459-33abdf80-3c21-11e9-873e-c022c8612351.png)
![typical-urial-173 jurassic ninja-05](https://user-images.githubusercontent.com/390760/53639460-33abdf80-3c21-11e9-9bec-97da46b13b6d.png)


#### After

![great-dolphins jurassic ninja-01](https://user-images.githubusercontent.com/390760/53639159-5ee1ff00-3c20-11e9-8e3c-51d1b4e1500d.png)
![great-dolphins jurassic ninja-02](https://user-images.githubusercontent.com/390760/53639160-5ee1ff00-3c20-11e9-9840-907551204a21.png)
![great-dolphins jurassic ninja-03](https://user-images.githubusercontent.com/390760/53639161-5ee1ff00-3c20-11e9-9a3d-e1ba4619977f.png)
![great-dolphins jurassic ninja-04](https://user-images.githubusercontent.com/390760/53639163-5f7a9580-3c20-11e9-98e9-7a2a624a90b0.png)
![great-dolphins jurassic ninja-05](https://user-images.githubusercontent.com/390760/53639164-5f7a9580-3c20-11e9-9ad0-81d6b9d851da.png)
